maskpostgresdata/management/commands/dump_masked_data.py

```diff
@@ -24,8 +23,8 @@ def add_arguments(self, parser):
             default=DEFAULT_DB_ALIAS,
         )
 
-    def filter_auth_user(self, queryset):
-        return queryset.update(password="blah")
+    def update_auth_user(self, queryset):
+        return queryset.update(password=make_password("password"))
```

Comment:
Maybe drop the `return` for this if it's not going to be used, it's just executed.